src/types/ImageStudio.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Reason for promise resolution
+ */
+export enum ImageStudioReason {
+  IMAGE,
+  CLOSED,
+}
+
+/**
+ * container interface for image representation
```

Comment:
```suggestion
 * Container interface for image representation
```
Capitalizing this would make it consistent with other docs, since they'll be visible to users of the library.